CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## Master (unreleased)
 
 * Find and remove 'Unplanned' labels on `cleanup-sprint`. Fixes #72.
+* Solve several rubocop offenses. Fixes #85, #86, #87, #88, #89.
```

Comment:
I don't think we need to mention this in the release notes. They are for the users of the app, so they don't care about development stuff, such as our linters configuration. You should add here feature and bug fixes.

Please remove it! :wink: